lib/modules/manager/pip_requirements/extract.ts

```diff
@@ -47,11 +47,9 @@ export function cleanRegistryUrls(registryUrls: string[]): string[] {
   });
 }
 
-export function extractPackageFile(
+export function extractPackageFileFlags(
```

Comment:
i would prefer to move this to a `common.ts` and directly import, so it's not a member of the public manager api.

Maybe we should do this as a refactor before this PR.

Reply:
Opened #29360